dev/x86_64/src/intt.S

```diff
@@ -256,6 +305,31 @@ vmovshdup	%ymm7,%ymm7
 vpblendd	$0xAA,%ymm8,%ymm6,%ymm6
 vpblendd	$0xAA,%ymm9,%ymm7,%ymm7
 
+/*
+ * The bound ceil(3q/4) for this scaling, as well as any other "Montgomery
+ * multiplication with signed canonical constant", is justified as follows.
+ *
+ * In Section 2.2 of https://eprint.iacr.org/2023/1962, they showed a bound that
+ * works for any variable input a, as long as the constant b is signed
+ * canonical:
+ *
+ *   |montmul(a, b)| <= (|a| (q/2) + (R/2) q) / R = (q/2) (1 + |a|/R).
+ *
+ * Therefore, even if we know nothing about a except that it fits inside
+ * int32_t (thus |a| <= R/2), we still have |montmul(a, b)| <= 3q/4. This can be
+ * strengthened to |montmul_pos(a, b)| <= floor(3q/4) < ceil(3q/4) since LHS is
+ * an integer and 3q/4 isn't.
+ *
+ * See test/test_bounds.py for more empirical evidence (and some minor technical
+ * details).
+ *
+ * TODO: Use proper citation. Currently, citations within asm can cause linter
+ *       to complain about unused citation, because comments are not preserved
+ *       after simpasm.
+ */
+
+/* 4, 5, 6, 7: abs bound < ceil(3q/4) */
```

Comment:
There are a lot of occurrences of `ceil(3q/4)` in the documentation. To reduce the amount of repetition, and seeing that we currently only rely on the output bound of Q, does it make sense to introduce a name for this here and argue only once what the actual value is?

```
In the following, we call MONTMUL_BOUND any absolute bound on the result of a signed Montgomery multiplication of a 16-bit signed integer with a signed canonical integer (in the range -Q/2...Q/2).
...
```